render/render-module.nix

```diff
@@ -173,21 +173,8 @@ in
               description = ''
                 Flake reference string that refers to the flake to import, used in the generated text for the installation instructions, see {option}`installation`.
               '';
-              default =
-                # This only works for github for now, but we can set a non-default
-                # value in the list just fine.
-                let
-                  match = builtins.match "https://github.com/([^/]*)/([^/]*)/blob/([^/]*)" config.baseUrl;
-                  owner = lib.elemAt match 0;
-                  repo = lib.elemAt match 1;
-                  # branch = lib.elemAt match 2; # ignored for now because they're all default branches
-                in
-                if match != null then
-                  "github:${owner}/${repo}"
-                else
-                  throw "Couldn't figure out flakeref for ${name}: ${config.baseUrl}";
```

Comment:
Removing the default for this means that in theory someone would have to provide it some day explicitly. But we don't have such a case in this branch, because all inputs are either `gitlab` or `github`, in which `flakeRef` is auto-resolved. And there's one explicit `flakeRef` case. I think this is not a problem.